tests/fixtures/rdf.py

```diff
@@ -0,0 +1,46 @@
+"""Test fixtures to set up a test named graph for loading RDF data."""
+from requests.auth import HTTPDigestAuth
+from flask import config
+import os
+import requests
+import pytest
+
+
+def get_sparql_auth_conf() -> dict:
+    """Fetch SPARQL auth configuration for the GN3_SECRETS file."""
+    sparql_conf = config.Config("")
+    sparql_conf.from_envvar("GN3_SECRETS")
```

Comment:
I think these should be reorderd i.e. GN3_CONF takes precedence over GN3_SECRETS